model/src/main/java/org/cbioportal/model/MutationalSignature.java

```diff
@@ -0,0 +1,56 @@
+package org.cbioportal.model;
```

Comment:
This class doesn't need to be in this PR - you have the other class in the core module.